mypy/util.py

```diff
@@ -639,8 +638,8 @@ def initialize_win_colors(self) -> bool:
         # Windows ANSI escape sequences are only supported on Threshold 2 and above.
         # we check with an assert at runtime and an if check for mypy, as asserts do not
         # yet narrow platform
-        assert sys.platform == "win32"
-        if sys.platform == "win32":
+        assert sys.platform == "win32", "Running not on Windows"
+        if sys.platform == "win32":  # needed to find win specific sys apis
```

Comment:
You can un-indent this.

Reply:
Nope, sadly I can't :(
Looks like that mypy does not narrow `sys.platform` after `assert` (this might be a bug).

Reply:
But, I added `assert False` in the end instead, it is more readable now.

Reply:
> Nope, sadly I can't :( Looks like that mypy does not narrow `sys.platform` after `assert` (this might be a bug).

Huh I was able to. But I guess that might just be because I was testing on Windows.

Reply:
Yes, you need to set `platform = win32` or `platform = linux` manually in `.ini` file for that.